pygfx/controllers/_panzoom.py

```diff
@@ -16,6 +17,9 @@ def __init__(
         up: Vector3 = None,
         zoom: float = 1.0,
         min_zoom: float = 0.0001,
+        scale: Vector3 = None,
```

Comment:
Now we have a scale and a zoom. I'd expect to have one vector that has the same values for each dimension when the aspect ratio should be maintained. Can we make the zoom a vector?

Reply:
ah, so we can have a zoom `(x, y z)` to replace scale